js/src/personal/bryan/About Me/AboutMeText.tsx

```diff
@@ -0,0 +1,17 @@
+import { Text, Space } from '@mantine/core'
+import classes from '../Bryan.module.css'
```

Comment:
nit: worth splitting up the main css module into different pages - especially in the next PR